hydra-plutus/src/Hydra/Contract/Head.hs

```diff
@@ -389,6 +390,10 @@ checkContest ctx contestationDeadline contestationPeriod parties closedSnapshotN
     && checkSignedParticipantContestOnlyOnce
     && mustBeWithinContestationPeriod
     && mustUpdateContesters
+    -- XXX: This check is redundant and can be removed,
```

Comment:
Why don't we remove it in this PR then? Or you wanted to make a separate one just for these XXX items?

Reply:
Yes we want to avoid changing the on chain scripts in this PR